### PR TITLE
NIAID POC calls Requestor

### PR DESCRIPTION
**Pull request by paulineribeyre**

### CODE REVIEW

File: src/StudyViewer/StudyDetails.jsx [hunk 1]
```diff
@@ -16,11 +18,27 @@ const onDownload = () => {
...
+      resource_path: '/programs/TODO',
+      resource_name: this.props.data.name,
+    };
+    fetchWithCreds({
+      path: `${requestorPath}request`,
+      method: 'POST',
+      body: JSON.stringify(body),
+    }).then(
+      ({ data }) => {
+        if (data && data.redirect_url) {
```
Comment (mfshao) [inline src/StudyViewer/StudyDetails.jsx:34]: do we need to do anything in case of this fails?

Comment (paulineribeyre) [reply src/StudyViewer/StudyDetails.jsx:34]: probably but i'm counting on you to handle it in the real code 😆

Comment (mfshao) [reply src/StudyViewer/StudyDetails.jsx:34]: yeah that's fine, i mean do they (niaid devs) need that when they are playing with this mock up?

Comment (paulineribeyre) [reply src/StudyViewer/StudyDetails.jsx:34]: i don't think so, i told them it's WIP

**Pull request merged**